packages/mock-server/src/data/data.ts

```diff
@@ -25,62 +25,67 @@ export const getStockLinesForItem = (
   return stockLines.filter(getFilter(item.id, 'itemId'));
 };
 
-const createStockLines = (items: Item[]) => {
+export const createStockLines = (
```

Comment:
Looks like a lot of changes, but the main changes in here are just to get the export into sqlite working:

- added stores
- a few date changes - expiry dates require a different date format
- invoice type needs to YELL
- pricing change